src/views/Picklist-Detail.vue

```diff
@@ -2,76 +2,182 @@
+          <ion-item-divider>
+            <ion-label> {{ picklist.alphabet }}</ion-label>
+          </ion-item-divider>
+          <PicklistDetailItem :picklists="picklist.record"/>
+        </ion-item-group>
       </ion-list>
-
-      <ion-fab vertical="bottom" horizontal="end" slot="fixed">
-        <ion-fab-button @click="presentAlertMultipleButtons">
-          <ion-icon :icon="checkmarkDone "/>
-        </ion-fab-button>
-      </ion-fab>
      </ion-content>
+
+     <ion-footer>
+      <ion-toolbar>
+        <ion-buttons class="footer-buttons">
+          <ion-button class="action-button" fill="outline" color="secondary" @click="scanCode()">
+            <ion-icon slot="start" :icon="barcodeOutline" />{{ $t("Scan") }}
+          </ion-button>
+          <ion-button class="action-button"  @click="completeProductPicklist" fill="outline" color="success">
+            <ion-icon slot="start" :icon="checkmarkDone" />{{ $t("Complete") }}
+          </ion-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-footer>
  </ion-page>
 </template>
 
-<script lang="ts">
-import { IonBackButton, IonButton, IonButtons, IonContent, IonFab, IonFabButton, IonHeader, IonIcon, IonItemDivider, IonList, IonLabel, IonPage, IonTitle, IonToolbar, alertController } from '@ionic/vue';
+<script>
+import { IonBackButton, IonButton, IonButtons, IonContent,IonFooter, IonHeader, IonIcon, IonItemDivider, IonItemGroup, IonLabel, IonList, IonPage, IonTitle, IonToolbar, alertController, modalController } from '@ionic/vue';
 import { defineComponent } from 'vue';
-import { checkmarkDone } from 'ionicons/icons';
+import { barcodeOutline,checkmarkDone } from 'ionicons/icons';
 import PicklistDetailItem from '@/components/Picklist-detail-item.vue';
+import { mapGetters, useStore } from 'vuex';
+import { translate } from '@/i18n'
+import { showToast } from '@/utils';
+import Scanner from '@/components/Scanner'
+import { useRouter } from 'vue-router';
+import { JsonCSV } from '@/mixins/jsonToCsv'
+import emitter from '@/event-bus'
 
 export default defineComponent({
-  name: 'Settings',
+  name: 'PicklistDetail',
   components: {
     IonBackButton, 
     IonButton, 
     IonButtons, 
-    IonContent, 
-    IonFab, 
-    IonFabButton, 
+    IonContent,
+    IonFooter, 
     IonHeader, 
-    IonIcon, 
-    IonItemDivider, 
-    IonList, 
-    IonLabel, 
+    IonIcon,
+    IonItemDivider,
+    IonItemGroup, 
+    IonLabel,
+    IonList,
     IonPage, 
     IonTitle, 
     IonToolbar,
     PicklistDetailItem
   },
+  computed: {
+    ...mapGetters({
+      picklists: 'picklist/getPicklists',
+      picklistItem: 'picklist/getCurrent',
+    })
+  },
+  data() {
+    return {
+      picklistGroup: []
+    }
+  },
+  props: ['id'],
+  mixins: [JsonCSV],
+  mounted () {
+    this.picklistItem.pickingItemList.sort((a, b) => a.productName.localeCompare(b.productName, 'es', { sensitivity: 'base' }));
```

Comment:
Please add comments here. What we are doing and why?

Reply:
Ok sir

Reply:
Done sir